src/test/java/uk/gov/hmcts/ccd/domain/service/createevent/CreateCaseEventServiceTest.java

```diff
@@ -98,6 +99,8 @@ class CreateCaseEventServiceTest extends TestFixtures {
     private CaseService caseService;
     @Mock
     private UserAuthorisation userAuthorisation;
+    @Mock
+    private TimeToLiveService timeToLiveService;
```

Comment:
Add atleast a verify that TTL service called for validateSuspensionChange & getUpdatedResolvedTTL.